src/fsharp/vs/ServiceLexing.fsi

```diff
@@ -15,6 +15,9 @@ open System.Collections.Generic
 open Microsoft.FSharp.Compiler 
 open Microsoft.FSharp.Compiler.Range
 
+type Position = int * int
```

Comment:
@smoothdeveloper  This is exposing the type that was defined in ServiceLexing.fs. No new behavior was introduced.